interpreter/cling/lib/Interpreter/NullDerefProtectionTransformer.cpp

```diff
@@ -107,6 +107,11 @@ namespace cling {
     }
 
     NodeContext VisitStmt(Stmt* S) {
+      for (auto I = S->child_begin(), E = S->child_end(); I != E; ++I) {
+        if (*I)
+          NodeContext nc = Visit(*I);
+      }
+
       return NodeContext(S);
```

Comment:
Given that we now always return an unchanged `NodeContext` - can't we just return nothing? Or do we ever care about the `NodeContext` that `Visit()` returns?